Plugins/Flow.Launcher.Plugin.Explorer/Search/Everything/EverythingAPI.cs

```diff
@@ -39,7 +39,7 @@ public enum StateCode
         /// Gets or sets a value indicating whether [match path].
         /// </summary>
         /// <value><c>true</c> if [match path]; otherwise, <c>false</c>.</value>
-        public static bool MatchPath
+        public static bool MatchPath // TODO these seem to be unused, and they're related exactly to the feature i want: make everything search through the whole path. How should these be used? Should they be removed instead? IMHO, it seems better to use full path search by default, so i'd remove this completely
```

Comment:
@taooceros  do you have any opinions here?

Reply:
Thanks! I don't notice it actually🤣. Do we want it to be default or as an option for user to change

Reply:
Actually I would say let's make it default true. I like that feature. Please use that variable though but just make it default true

Reply:
Guide me here please:
> Actually I would say let's make it default true. I like that feature. Please use that variable though but just make it default true

what do you mean by this?

Moreover, something is unclear to me: why are all those variables statics? Shouldn't there be only 1 instance of `EverythingAPI` (shouldn't `EverythingAPI` be a singleton)?

Reply:
Sorry don't care about it.

Static class is a singleton in some way but if you want to make it singleton I would be happy to merge.

Reply:
For now, i think we can remove those static completely, as they're unused. I also pushed a commit which adds a flag to `EverythingSearchOption` about full path search. I think it's better than to have it hardcoded

Reply:
I've gone ahead and removed them.